docs/en/Tiered-Storage-on-Alluxio.md

```diff
@@ -1,88 +1,117 @@
-the file size exceeds the size of the top tier, the write will also fail.
+```
+alluxio.worker.tieredstore.level0.dirs.path=/mnt/ramdisk,/mnt/ssd1,/mnt/ssd2
+```
 
-The user can also specify the tier that the data can be written to via
-[configuration settings](#configuration-parameters-for-tiered-storage).
+After updating the storage media, we need to indicate how much storage is allocated for each storage
+directory. For example, if we wanted to use 16 GB on the ramdisk and 100 GB on each SSD:
 
-Reading data with the ReadType.CACHE or ReadType.CACHE_PROMOTE will also result in the data being
-written into Alluxio. In this case, the data is always written to the top tier.
+```
+alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
+```
 
-Finally, data in written into Alluxio via the load command. In this case also, the data is always
-written to the top tier.
+Note that the ordering of the quotas much match with the ordering of the paths.
 
-### Reading Data
+## Eviction
 
-Reading a data block with tiered storage is similar to standard Alluxio. If the data is already in
-Alluxio will simply read the block from where it is already stored. If Alluxio is configured with
-multiple tiers, then the block will not be necessarily read from the top tier, since it could have
-been moved to a lower tier transparently.
+Because Alluxio storage is designed to be volatile, there must be a mechanism to make space for new
+data when Alluxio storage is full. This is termed eviction.
 
-Reading data with the ReadType.CACHE_PROMOTE will ensure the data is first transferred to the
-top tier before it is read from the worker. This can also be used as a data management strategy by
-explicitly moving hot data to higher tiers.
+There are two modes of eviction in Alluxio, asynchronous (default) and synchronous. You can switch
+between the two by enabling and disabling the space reserver which handles asynchronous eviction.
+For example, to turn off asynchronous eviction:
+
+```
+alluxio.worker.tieredstore.reserver.enabled=false
+```
+
+Asynchronous eviction relies on a periodic space reserver thread in each worker to evict data. It
+waits until the worker storage utilization reaches a configurable high watermark. Then it evicts
+data based on the eviction policy until it reaches the configurable low watermark. For example, if
+we had the same 16+100+100=216GB storage configured, we can set eviction to kick in at around 200GB
+and stop at around 160GB:
+
+```
+alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9
+alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75
+```
 
-#### Pinning Files
+Synchronous eviction is the legacy implementation of eviction. It waits for a client to request more
+space than is currently available on the worker and then kicks off the eviction process to free up
+enough space to serve that request. This leads to many small eviction attempts, which is less
+efficient.
```

Comment:
mentioning this is not recommended to use?

Reply:
Updated, thanks.